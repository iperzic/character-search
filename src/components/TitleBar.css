.TitleBar {
    padding-bottom: 5px;
}
