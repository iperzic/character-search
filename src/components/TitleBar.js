import React from 'react';
import PropTypes from 'prop-types';

import './TitleBar.css';

const TitleBar = ({ text }) => (
  <div className="TitleBar">
    {text}
  </div>
);

TitleBar.propTypes = {
  text: PropTypes.string.isRequired,
};

export default TitleBar;
