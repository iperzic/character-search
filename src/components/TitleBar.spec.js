import React from 'react';
import { shallow } from 'enzyme';

import TitleBar from './TitleBar';

describe('TitleBar', () => {
  let component;
  let text;

  beforeEach(() => {
    text = 'some text';
    component = shallow(<TitleBar text={text} />);
  });

  it('should show provided text', () => {
    expect(component.getElement()).toMatchSnapshot();
  });
});
