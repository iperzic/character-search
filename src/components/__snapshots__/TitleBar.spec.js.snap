// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleBar should show provided text 1`] = `
<div
  className="TitleBar"
>
  some text
</div>
`;
